tests/e2e/framework/util.go

```diff
@@ -49,7 +49,11 @@ var RunID = uuid.NewUUID()
 
 // RandomSuffix provides a random sequence to append to pods,services,rcs.
 func RandomSuffix() string {
-	return strconv.Itoa(rand.Intn(10000))
+	nBig, err := rand.Int(rand.Reader, big.NewInt(10000))
```

Comment:
https://github.com/kubernetes/kubernetes/blob/2183a84feb564582489d56c6570917959f003726/staging/src/k8s.io/apimachinery/pkg/util/rand/rand.go#L98

Reply:
That is using `math/rand`, not `crypto/rand`. Although I agree it's a good way of circumventing the golangci-lint check :P

Reply:
The intent is not to circumvent the checks. The intent is to find a simpler way to generate a unique suffix without having to "roll-our-own" implementation all the time.

I am also not concerned about using weak random number generators in our tests since we do not require them to be cryptographically secure. We just require a lower chance of name collisions, and at the scale we're operating, these are already low.

Reply:
I understand. I have addressed this comment and updated the PR. The diff should look a lot more simpler now